### PR TITLE
[4.0] Installation language

### PR DESCRIPTION
**Pull request by brianteeman**
Remove hard coded string and replace with translatable string

Add new string for the site name section - wrong string was being used

Ensure all section strings are consistent Capital case

Update en-US translation to be in sync with en-GB

### CODE REVIEW

**Review by mbabker — APPROVED** [review]

File: installation/language/en-US/en-US.ini [hunk 1]
```diff
@@ -29,28 +30,33 @@ INSTL_PRECHECK_ACTUAL="Actual"
 ; Database view
 INSTL_DATABASE="Database Configuration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL database query failed."
-INSTL_DATABASE_HOST_DESC="Specify the Host Name, usually \"localhost\" or a name provided by your host."
+INSTL_DATABASE_HOST_DESC="Enter the Host Name, usually \"localhost\" or a name provided by your host."
 INSTL_DATABASE_HOST_LABEL="Host Name"
-INSTL_DATABASE_NAME_DESC="Specify the database table name. Use table prefix to distinct Joomla sites."
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named &quot;%1$s&quot; and upload it to the &quot;%2$s&quot; folder of your Joomla site."
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="In order to confirm that you are the owner of this website please delete the file named &quot;%1$s&quot; we have created in the &quot;%2$s&quot; folder of your Joomla site."
```
Comment (Quy) [inline installation/language/en-US/en-US.ini]: Change `&quot;` to `\"`

File: installation/language/en-US/en-US.ini [hunk 2]
```diff
@@ -29,28 +30,33 @@ INSTL_PRECHECK_ACTUAL="Actual"
 ; Database view
 INSTL_DATABASE="Database Configuration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL database query failed."
-INSTL_DATABASE_HOST_DESC="Specify the Host Name, usually \"localhost\" or a name provided by your host."
+INSTL_DATABASE_HOST_DESC="Enter the Host Name, usually \"localhost\" or a name provided by your host."
```
Comment (Quy) [inline installation/language/en-US/en-US.ini]: Make lowercase `Host Name` to  be consistent with `Enter the database name`

File: installation/language/en-US/en-US.ini [hunk 3]
```diff
@@ -30,10 +30,10 @@ INSTL_PRECHECK_ACTUAL="Actual"
 ; Database view
 INSTL_DATABASE="Database Configuration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL database query failed."
-INSTL_DATABASE_HOST_DESC="Enter the Host Name, usually \"localhost\" or a name provided by your host."
+INSTL_DATABASE_HOST_DESC="Enter the host name, usually \"localhost\" or a name provided by your host."
 INSTL_DATABASE_HOST_LABEL="Host Name"
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named &quot;%1$s&quot; and upload it to the &quot;%2$s&quot; folder of your Joomla site."
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="In order to confirm that you are the owner of this website please delete the file named &quot;%1$s&quot; we have created in the &quot;%2$s&quot; folder of your Joomla site."
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named /"%1$s/" and upload it to the /"2$s/" folder of your Joomla site."
```
Comment (Quy) [inline installation/language/en-US/en-US.ini]: Should  be `\"`

File: installation/language/en-GB/en-GB.ini [hunk 4]
```diff
@@ -32,8 +32,8 @@ INSTL_DATABASE="Database Configuration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL database query failed."
 INSTL_DATABASE_HOST_DESC="Enter the host name, usually \"localhost\" or a name provided by your host."
 INSTL_DATABASE_HOST_LABEL="Host Name"
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named  /"%1$s/" and upload it to the  /"%2$s/" folder of your Joomla site."
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="In order to confirm that you are the owner of this website please delete the file named  /"%1$s/" we have created in the  /"%2$s/" folder of your Joomla site."
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named  \"%1$s\" and upload it to the  \"%2$s\" folder of your Joomla site."
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini]: Remove extra space before `\"`. 😮

**Comment by Quy** [comment]
Please change `customise` https://github.com/joomla/joomla-cms/pull/20515/files#diff-08b8aee1d43899c68aa8a784f98becb5R85

File: installation/language/en-GB/en-GB.ini [hunk 5]
```diff
@@ -12,6 +12,7 @@ MIN_PHP_ERROR_URL_TEXT="Help me resolve this"
 ;End of minimum PHP error page
 
 ;Main Config
+INSTL_SELECT_INSTALL_LANG="Select Installation Language"
 INSTL_SELECT_LANGUAGE_TITLE="Select Language"
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini:16]: The other labels are in sentence form. Lets make this one too. `Select language.`

File: installation/language/en-GB/en-GB.ini [hunk 6]
```diff
@@ -88,11 +90,11 @@ INSTL_COMPLETE_SITE_BTN="Complete & Open Site"
 INSTL_COMPLETE_ADMIN_BTN="Complete & Open Admin"
 INSTL_COMPLETE_FINAL="Installation is Complete"
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini:95]: Can we change this to `Installation Completed` or `Installation Done`?

**Comment by wilsonge** [comment]
LGTM - are you planning on addressing the other two issues @Quy raised @brianteeman ? They're slightly less related to the main PR so I'm easy either way - just don't want to prematurely merge :)

File: installation/language/en-GB/en-GB.ini (hunk 6 above)
Comment (brianteeman) [reply installation/language/en-GB/en-GB.ini:95]: I really dont like that word

**Comment by infograf768** [comment]
Not sure we should merge this now as we have quite a few code descrepancies vs lang strings that we should solve altogether if possible.

Examples:
`<h3><?php echo JText::_('Do you want to install sample data?'); ?></h3>` // We DO need a lang string for that one.
and
`$setting->label = \JText::_('INSTL_MAGIC_QUOTES_RUNTIME');` // string missing but do we use it?

File: installation/language/en-GB/en-GB.ini (hunk 5 above)
Comment (brianteeman) [reply installation/language/en-GB/en-GB.ini:16]: actually the other labels are sentences - this one is not. I am happy with this one as is

**Comment by brianteeman** [comment]
@infograf768 think I have caught all the missing strings now.

**Comment by infograf768** [comment]
INSTL_MAGIC_QUOTES_RUNTIME is missing when we look at code.
INSTL_MAGIC_QUOTES_RUNTIME="Magic Quotes Runtime"

**Comment by mbabker** [comment]
In the context of 4.0 there really should be no references to Magic Quotes Runtime.  PHP 5.4 removed it.

**Comment by brianteeman** [comment]
That is really beyond the scope of the intent of this PR but i have removed it

**Comment by mbabker** [comment]
It only caught my eye because it was pointed out the string was missing.

**Comment by infograf768** [comment]
Please also take it off from UpdateModel.php so that we are done with it.

**Comment by brianteeman** [comment]
@infograf768 already done in a seperate pr

File: installation/src/Model/ChecksModel.php [hunk 7]
```diff
@@ -188,13 +188,6 @@ public function getPhpSettings()
 		$setting->recommended = true;
 		$settings[] = $setting;
 
-		// Check for magic quotes runtimes.
```
Comment (Quy) [inline installation/src/Model/ChecksModel.php:191]: Revert? Already in #20555.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**